src/app/core/services/mocks.service.ts

```diff
@@ -166,20 +166,73 @@ export class MocksService {
     });
   }
 
-  Users ({limit, offset}) {
-    return {
-        table: {
-            count: 3
-        },
-        data: times(3, (index) => {
+    Users ({limit, offset}) {
+        return {
+            table: {
+                count: 3
+            },
+            data: times(3, (index) => {
+                return {
+                    'id' : index + +offset + 1,
+                    'firstname': faker.name.findName().split(' ')[0],
+                    'lastname': faker.name.findName().split(' ')[0],
+                    'email': faker.internet.email()
+                };
+            })
+        };
+    }
+
+    createVPC (form: IVPCInformation): IResponse {
+        console.log('Creating vpc: ' , form);
```

Comment:
I thing don't need to show form data in console, if there is a message, it's enough